src/guides/v2.3/javascript-dev-guide/javascript/requirejs.md

```diff
@@ -62,6 +62,28 @@ paths: {
 }
 ```
 
+Let us consider an example of overwriting an HTML file in the adminhtml.
+In this example, the max-length value of the text-box in the adminhtml is altered. The HTML file is at `vendor/magento/module_ui/view/base/web/templates/form/element/input.html`
+
+-Create a requirejs-config.js file under `app/code/<Vendor_Name>/<Module_Name>/view/base/`
```

Comment:
You need correct spacing after the - otherwise the bullet list is not created correctly. This is throughout all your PR

Reply:
The linter does not allow a space after '-' @BarnyShergold

Reply:
It's a double space after the - 
See : https://github.com/magento/devdocs/pull/8348/files (fastly-vcl-badreferer.md)

Reply:
Got it! Thanks! @BarnyShergold